lgi/lgi.h

```diff
@@ -29,6 +29,29 @@
 #endif
 #endif
 
+/* Lua 5.3 integers. */
+#if LUA_VERSION_NUM >= 503
+// For Lua 5.3
+typedef LUA_UNSIGNED lgi_Unsigned;
+#if LUA_INT_TYPE == LUA_INT_INT
+#define LGI_LUAINT_FORMAT "%d"
+#elseif LUA_INT_TYPE == LUA_INT_LONG
+#define LGI_LUAINT_FORMAT "%ld"
+#else
+#define LGI_LUAINT_FORMAT "%lld"
+#endif
```

Comment:
All of these `#ifdef`s can be replaced by just using `LUA_INTEGER_FMT`, I think (but haven't checked).

Reply:
Going to check on Lua 5.1/5.2.